pnpm-lock.yaml

```diff
@@ -5078,7 +5078,7 @@ packages:
   /@types/concat-stream/1.6.1:
     resolution: {integrity: sha512-eHE4cQPoj6ngxBZMvVf6Hw7Mh4jMW4U9lpGmS5GBPB9RYxlFg+CHaVN7ErNY4W9XfLIEn20b4VDYaIrbq0q4uA==}
     dependencies:
-      '@types/node': 8.10.66
+      '@types/node': 17.0.18
```

Comment:
how can `@types/concat-stream` update its dependency when its version is unchanged? 👀

Reply:
@sheremet-va i was wondering too because i only ran `pnpm i` and got that updated